Yank/experiment.py

```diff
@@ -207,7 +328,38 @@ def initialize_alchemical_phase(self):
 
 
 class Experiment(object):
-    """An experiment built by ExperimentBuilder."""
+    """
+    An experiment built by ExperimentBuilder.
+
+    This is a completely defined experiment with all parameters and settings ready to go.
+
+    It is highly recommended to **NOT** use this class directly, and instead rely on the ExperimentBuilder class to
+    parse all options, configure all phases, properly set up the experiments, and even run them.
+
+    These experiments are frequently created with the
+
+    Parameters
+    ----------
+    phases : list of yank.yank.AlchemicalPhases
+        Phases to run for the experiment
+    number_of_iterations : int
+        Total number of iterations each phase will be run for
+    switch_phase_interval : int
+        Number of iterations each phase will be run before the cycling to the next phase
+
+    Attributes
+    ----------
+    iteration
+
+    Methods
+    -------
+    run
```

Comment:
I don't think the [numpy docstring convention](https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#documenting-classes) includes a `Methods` section, since the methods themselves provide this list.